src/components/panels/details/info-multiple-selection.tsx

```diff
@@ -51,16 +70,140 @@ export default function InfoMultipleSection() {
         }
     };
 
+    const [filterNodes, setFilterNodes] = React.useState(true);
+    const [filterEdges, setFilterEdges] = React.useState(true);
+    React.useEffect(() => {
+        setFilterNodes(true);
+        setFilterEdges(true);
+    }, [selected]);
+
+    const [isOpenChangeModal, setIsOpenChangeModal] = React.useState(false);
+
+    return (
+        <>
+            <Box>
+                <Heading as="h5" size="sm">
+                    {t('panel.details.multipleSelection.selected')} {selected.size}
+                </Heading>
+                <VStack m="var(--chakra-space-1)">
+                    <HStack m="var(--chakra-space-1)" width="100%">
+                        <Button
+                            size="sm"
+                            width="100%"
+                            variant={filterNodes ? 'solid' : 'ghost'}
+                            onClick={() => setFilterNodes(!filterNodes)}
+                            colorScheme="blue"
+                        >
+                            {t('panel.details.multipleSelection.filterNodes')}
+                        </Button>
+                        <Button
+                            size="sm"
+                            width="100%"
+                            variant={filterEdges ? 'solid' : 'ghost'}
+                            onClick={() => setFilterEdges(!filterEdges)}
+                            colorScheme="blue"
```

Comment:
This might not be a good UX design as blue buttons are generally linked to default or recommended actions which is not the case here. A checkbox or RmgButtonGroup should be used :)